Contributing.md

```diff
@@ -0,0 +1,50 @@
+# Stakeholders
```

Comment:
Perhaps I'm showing my lack of knowledge in the English language, but as stakeholder I'd be thinking about people who has invested money in the project. When we do crowd funding, that would be the people contributing to the crowd funding.
I'd suggest to rename "Stakeholders" with "Roles", since what I see is a list of roles you can have within the project.

Reply:
Good suggestion, and you're right that the name "stakeholder" does better fit the definition you described.